healthcareai/common/filters.py

```diff
@@ -72,6 +72,11 @@ def transform(self, x, y=None):
         validate_dataframe_input(x)
 
         subset = [c for c in x.columns if c not in self.excluded_columns]
+
         x.dropna(axis=0, how='any', inplace=True, subset=subset)
 
+        if x.empty:
+            raise HealthcareAIError(
+                'No rows left in the DataFrame after pipeline due to NaN values located in every row')
```

Comment:
Excellent. Let's make the error even a little more user friendly and descriptive. Something like:
"Because imputation is set to False, rows with missing or null/NaN values are being dropped. In this case, all rows contain null values and therefore were ALL dropped. Please consider using imputation or assessing the data quality and availability"

I'm totally open to making this simpler and more educational.